aws/utils_test.go

```diff
@@ -0,0 +1,52 @@
+package aws
```

Comment:
Could you please remove this test file? Having unit test cases is great, but I believe it may not be necessary in this case.